english/alchemy/alchemy.md

```diff
@@ -2,7 +2,7 @@ You take off your backpack and pull out your trusty acetylene torch. You place t
 marshmallow in a crucible and apply the flame. Chanting an incantation that has been
 passed down through generations of alchemists, you stare into the glowing crucible as the
 marshmallow melts into a cauldron of orange flame. As the fiery light coalesces into the
-shape of a coin at the bottom of the crucible, you turn off the flame. You remove a 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.